db/patterns/videobot.eno

```diff
@@ -0,0 +1,12 @@
+name: Videobot
+category: audio_video_player
+website_url: https://www.videobot.com/
+organization: videobot
+
+--- domains
+videobot.com
+--- domains
+
+--- filters
+||videobot.com^$3P
```

Comment:
```suggestion
||videobot.com^$3p
```

Not clear from the test failure error, but it looks like the parser is quite picky here. "3p" has be lower case.